.storybook/stories/LendCtaExp.stories.js

```diff
@@ -13,17 +13,6 @@ const queryResult = {
 	}
 };
 
-const queryResultLess25 = {
```

Comment:
It looks like the other query result objects can be removed as well.

Reply:
Fixed